OpenRA.Game/Server/ServerOrder.cs

```diff
@@ -42,6 +42,10 @@ public static ServerOrder Deserialize(BinaryReader r)
 						return new ServerOrder(name, data);
 					}
 
+				case 0xbf:
+					Console.WriteLine("Connection Terminated - This isn't a server order.");
```

Comment:
Can we make this message consistent with the `0xff` case?

Reply:
In that case, I'll move it to execute with 0xff.